### ISSUE TITLE
A11y : Cannot select library panel

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46489]
**Steps to reproduce**:

Try selecting a library panel using only a keyboard

**Actual Result**:

Cannot select library panel

**Expected Result**

keyboard-only workflow available

**Relevant WCAG Criteria:** 

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
